packages/lambdas/src/sqs-to-converter.ts

```diff
@@ -278,7 +273,6 @@ export async function handler(event: SQSEvent) {
           patientId,
           s3FileName,
           updatedConversionResult,
-          jobStartedAt,
```

Comment:
Likely this is being used somewhere in analytics so has probably been broken since we stopped processing -- we should likely check.